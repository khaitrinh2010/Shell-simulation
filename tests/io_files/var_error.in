var %invalid*name Messi 
var -z name khai  
var messi ronaldo goat
