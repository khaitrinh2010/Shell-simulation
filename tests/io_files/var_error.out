>> var: invalid characters for variable %invalid*name
>> var: invalid option: -z
>> var: expected 2 arguments, got 3
>> 
